PowerEditor/installer/nativeLang/chineseSimplified.xml

```diff
@@ -970,17 +1043,17 @@
                 <Item id="7004" name="文件排序(&amp;T)"/>
             </Window>
             <ColumnEditor title="列编辑">
-                <Item id="2023" name="插入文本"/>
-                <Item id="2033" name="插入数字"/>
-                <Item id="2030" name="初始值 :"/>
-                <Item id="2031" name="增量 :"/>
-                <Item id="2035" name="开头补零"/>
-                <Item id="2036" name="重复次数 :"/>
-                <Item id="2032" name="格式"/>
-                <Item id="2024" name="十进制"/>
-                <Item id="2025" name="八进制"/>
-                <Item id="2026" name="十六进制"/>
-                <Item id="2027" name="二进制"/>
+                <Item id="2023" name="插入文本(&amp;T)"/>
+                <Item id="2033" name="插入数字(&amp;N)"/>
+                <Item id="2030" name="初始值(&amp;I) :"/>
+                <Item id="2031" name="增量(&amp;Y) :"/>
+                <Item id="2035" name="开头补零(&amp;Z)"/>
+                <Item id="2036" name="重复次数(&amp;R) :"/>
+                <Item id="2032" name="格式(&amp;F)"/>
```

Comment:
Sorry I just fixed an issue regarding it: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
`&amp;F` should be removed.

Reply:
@donho new changes in 55d6717 are added.